apps/airdrop/src/App/AirdropConfirmation.tsx

```diff
@@ -0,0 +1,274 @@
+  accountType: string;
+  source: string;
+  category: string;
+  minNam: number;
+};
+
+export const AirdropConfirmation: React.FC = () => {
+  const iconsContainerRef = useRef<HTMLDivElement>(null);
+  const [confirmation] = useAtom(confirmationAtom);
+  const [totalMinNam, setTotalMinNam] = useState<number>();
+  const [breakdown, setBreakdown] = useState<Breakdown[]>([]);
+
+  // We want to scroll to the top of the page when the component is mounted
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, []);
+
+  if (confirmation === null) {
+    throw new Error("Confirmation state is empty!");
+  }
+
+  useEffect(() => {
+    (async () => {
+      const { ok, result } = await getAllClaims(confirmation.address);
+      if (ok) {
+        const totalMinNam = result.claims.reduce((acc, curr) => {
+          return acc + curr.token;
+        }, 0);
+        setTotalMinNam(totalMinNam);
+
+        const breakdown = result.claims.map((claim) => {
+          return {
+            accountType: categoryAccountTypeMap[claim.category],
+            source: claim.value,
+            category: getCategory(claim.category, claim.eligible_for),
+            minNam: claim.token,
+          };
+        });
+        setBreakdown(breakdown);
+      }
+    })();
+  }, [confirmation.address]);
+
+  useLayoutEffect(() => {
+    if (!iconsContainerRef.current) return;
+    return iconsOnMouseMovement(iconsContainerRef.current);
+  }, []);
+
+  return (
+    <AirdropConfirmationContainer>
+      <GlobalStyles colorMode="light" />
+      <AirdropConfirmationMainSection>
+        <AirdropConfirmationSection>
+          <Stack gap={5}>
+            <Stack gap={2}>
+              <AirdropConfirmationHeading
+                level={"h1"}
+                size={"6xl"}
+                themeColor={"utility1"}
+              >
+                Namada
+                <br />
+                Genesis account
+                <br />
+                submitted
+              </AirdropConfirmationHeading>
+              <Text themeColor={"utility1"}>
+                NAM will be available diretly in your wallet
+                <br /> at Namada Mainnet launch, subject to the
+                <br />{" "}
+                <LinkButton themeColor="utility1">
+                  <b>terms of Service</b>
+                </LinkButton>
+              </Text>
+            </Stack>
+            <Stack gap={3}>
+              <AirdropConfirmationInput
+                label="Genesis public key:"
+                variant={InputVariants.ReadOnlyCopy}
+                value={confirmation.publicKey}
+              />
+              <AirdropConfirmationInput
+                label="Genesis transparent account:"
+                variant={InputVariants.ReadOnlyCopy}
+                value={confirmation.address}
+              />
+            </Stack>
+
+            <Stack gap={"px"}>
+              <Heading themeColor={"utility1"} level={"h4"} size={"xl"}>
+                Minimum NAM claimed
```

Comment:
I am not understanding the MIN concept here? As I understand it, it is an estimate of quantity, right?